www/source/docs/reference/plan-syntax.html.md

```diff
@@ -255,6 +255,10 @@ do_download()
 do_verify()
 : The default implementation tries to verify the checksum specified in the plan against the computed checksum after downloading the source tarball to disk. If the specified checksum doesn't match the computed checksum, then an error and a message specifying the mismatch will be printed to stderr. You should not need to override this behavior unless your package does not download any files.
 
+do_check()
+: The default implementation runs post-compile and checks provided two conditions
```

Comment:
I don't think this is clear. The default implementation does nothing, and you're expected to put something like `make test`, or whatever runs your plan's tests after it's built, in it.

These listed in the order they are run, so `do_check` should go between `do_build` and `do_install`.